lib/utils/import/README.MD

```diff
@@ -0,0 +1,30 @@
+# Simplenote Import
+
+This is the core importer for Simplenote. It expects a JSON object of notes in the same format as the `import` module. It is aggressive about only allowing certain properties, and converts dates into timestamps where appropriate. Importing specific export files (Such as Evernote .enex) should use a separate conversion script and then pass the resulting JSON object to `importNotes()`.
+
+Example JSON:
```

Comment:
Maybe the “Example JSON” should be “Example JSON with allowed properties”, and then remove the `id` prop?